rpc/rpc_test.go

```diff
@@ -572,13 +572,13 @@ func (root *Root) assertServerNotified(c *gc.C, p testCallParams, requestId uint
 		c.Assert(serverReply.body, gc.Equals, stringVal{p.request().Action + " ret"})
 	}
 	if p.retErr && p.testErr {
-		c.Assert(serverReply.hdr, gc.Equals, rpc.Header{
+		c.Assert(serverReply.hdr, gc.DeepEquals, rpc.Header{
```

Comment:
The jc.DeepEquals gives nicer error output I believe.

Reply:
Ah, didn't know that was a thing! I will push a commit to switch to that.